isis/src/mex/apps/hrsc2isis/MexHrscKernels.trn

```diff
@@ -0,0 +1,21 @@
+# Translations for HRSC Stereo and SRC instrument.
+# ISIS cube label Kernels group
+
+Group = NaifIkCode
+  Auto
+  InputKey       = DETECTOR_ID
+  OutputName     = DetectorId
+  OutputPosition = (Group, Archive)
```

Comment:
Archive -> Kernels. `spiceinit` did not like this.

Reply:
thanks